Submitty/Submitty
# [Bugfix:Submission] Fix Notebook Display bugs

### What is the current behavior?
closes #5633 

### What is the new behavior?
1. Students can see their notebook after submissions are closed even if accept late submissions=no
2. Students with extensions will be able to see and submit their notebook if submissions are closed.
3. if a student has an extension but accept late submissions = no, they still won't be able to submit
4. A nicer (not red) message is displayed if students are trying to check their grade on a notebook and accept late submissions=no

### Other information?

### NOTE:
you may run into multiple twig exceptions when testing, I am going to go through and fix them but if you test this before that happens, just delete the lines throwing exceptions because they are irrelevant to this pr.

To test:
1. Close a notebook gradeable in the dev course
2. toggle accept late submissions and see different behavior 
3. give students extensions and see behavior
